sql/atxs/atxs.go

```diff
@@ -73,6 +72,29 @@ func GetTimestamp(db sql.Executor, id types.ATXID) (timestamp time.Time, err err
 	return timestamp, err
 }
 
+// GetFirstIDByNodeID gets the initial ATX ID for a given node ID.
+func GetFirstIDByNodeID(db sql.Executor, nodeID types.NodeID) (id types.ATXID, err error) {
+	enc := func(stmt *sql.Statement) {
+		stmt.BindBytes(1, nodeID.ToBytes())
+	}
+	dec := func(stmt *sql.Statement) bool {
+		stmt.ColumnBytes(0, id[:])
+		return true
+	}
+
+	if rows, err := db.Exec(`
+		select id from atxs 
+		where smesher = ?1
+		order by epoch asc, timestamp asc
```

Comment:
ordering by timestamp seems redundant. we simply don't care. for 2 reasons:
- timestamp is when the node received this ATX, this can be via gossip or via sync (much later than the ATX's actual gossip time)
- there is just 1 atx per smesher per epoch. 

so `order by epoch asc limit 1` is sufficient